woocommerce/woocommerce
# issue-35238 - fixing issue with getting mysql version when using mariadb

### All Submissions:






### Changes proposed in this Pull Request:



Improves how we determine the database server version, for improved compatibility with MariaDB.

Closes #35238.






### How to test the changes in this Pull Request:



1. Check out this branch, and navigate to the **WooCommerce ▸ Status** screen.
2. Scroll to the **Server environment** section, and look for **MySQL version**.
3. You should see a version number next to this, that corresponds with your test environment:

<div align="center"><img width="700" alt="Screenshot 2023-09-26 at 14 35 14" src="https://github.com/woocommerce/woocommerce/assets/3594411/9d942c9b-184a-42ca-90b2-50f215f15076"></div>

💡 If you are able to do so, you may wish to test this first using MySQL and second using MariaDB.



### Other information:




### FOR PR REVIEWER ONLY:
